torch/ao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -19,13 +19,13 @@ def generate_numeric_debug_handle(graph_module: GraphModule) -> None:
     """Attach numeric_debug_handle_id for all nodes in the model except for placeholder node
     The graph nodes of input model is modified inplace.
     """
-    unique_id = -1
+    unique_id = 0
```

Comment:
Is the goal here to have the first ID start at 1 instead of 0? Why? There's no summary on this PR